### ISSUE TITLE
Facet / sort by date (i.e. new publications)

### ISSUE THREAD

**Issue by shanitajohn** [mlibrary/heliotrope#31]
**STORY**
As a user when I view search results, I should be able to select "date" from the facet sidebar and see a list of dates representative of the content in the repository. When I click on a date I should see all the monographs associated with this date.  

**ACCEPTANCE CRITERIA**
- [ ] The publication date is listed in the facet sidebar
- [ ] When I click on a publication date from the facet sidebar I only see monographs that have this publication date
- [ ] When I click on a publication date from the facet sidebar I should not see monographs that do not have that publication date

**Comment by mark-dce** [comment]
Blocked by #200 (i.e. dates have to be guaranteed to be available in some normalized format)

**Comment by sethaj** [comment]
I think this can be closed. We have a "date" facet, or "publication year" for file_sets within a monograph on the monograph_catalog page which I think is really what we wanted. This ticket talks about date faceting for monographs, but we don't have enough monographs yet (and won't for some time) to really have any kind of valid use case for doing this work. #200 has been closed since June and it's now January.

When we get to the point of  needing date facets for monographs we'll make a new ticket.
